frontend/src/features/projects/common/projectConverter.ts

```diff
@@ -19,6 +19,28 @@ export const getCurrentFiscal = (fiscals: IFiscal[], key: FiscalKeys) => {
   return _.find(fiscals, { fiscalYear: currentFiscal, key: key });
 };
 
+const currentYear = moment().year();
+
+/**
+ * Get the most recent evaluation matching the current year and passed evaluation type.
+ * @param evaluations a list of evaluations belonging to this project
+ * @param key only return evaluations matching this key
+ */
+export const getCurrentYearEvaluation = (
```

Comment:
Fiscals were already doing this, and appraisals have their own logic, but assessed property values should be getting values from the current year.